src/component/variantPage/BasicInfo.tsx

```diff
@@ -319,6 +321,30 @@ export default class BasicInfo extends React.Component<IBasicInfoProps> {
         );
     }
 
+    private jsonButton() {
+        return (
+            <DefaultTooltip
+                placement="top"
+                overlay={
+                    <span>Click to read more about the raw API response</span>
```

Comment:
Maybe something like:
```suggestion
                    <span>
                            Click to view to raw API response<br /><br />
                            Click <a href="https://docs.genomenexus.org/api">here</a> for more info about the API 
                   </span>
```